app/components/homepage-components/TestimonialCard.tsx

```diff
@@ -12,7 +12,7 @@ interface TestimonialCardProps {
 
 const TestimonialCard = ({ testimonial }: TestimonialCardProps) => {
   return (
-    <div className="flex max-w-xl shrink-0 grow-0 basis-full flex-col p-2 lg:max-w-5xl lg:flex-row">
+    <article className="flex max-w-xl shrink-0 grow-0 basis-full flex-col p-2 lg:max-w-5xl lg:flex-row">
```

Comment:
I love the HTML Semantices!

Reply:
I'm trying to get out of Divtown and move to Semantic City